<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgImportPlayer</class>
 <widget class="QDialog" name="DlgImportPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter search string:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearchString"/>
     </item>
     <item>
      <widget class="QListWidget" name="lwNames"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnImport">
         <property name="text">
          <string>Import selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgImportPlayer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImport</sender>
   <signal>clicked()</signal>
   <receiver>DlgImportPlayer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwNames</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>DlgImportPlayer</receiver>
   <slot>onNameListSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSearchString</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgImportPlayer</receiver>
   <slot>onSearchStringChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwNames</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>DlgImportPlayer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNameListSelectionChanged()</slot>
  <slot>onSearchStringChanged()</slot>
 </slots>
</ui>
